economics/staking_protocol/main.tex

```diff
@@ -0,0 +1,355 @@
+The dramatic difference in long-term  cumulative earnings between various staking strategies over both phases is shown in figures 4 and 5. 
+
+\subsubsection{Winding down sub-stakes}
+
+The default setting for all sub-stakes is for $t_i$, the stake unlock duration, to be automatically prolonged. This keeps the subsidy coefficient ($\kappa$) constant. If a service-provider proactively initiates a wind-down – that is, they specify an unlock date in the future for a sub-stake, then the corresponding subsidy will continually decrease as the time to unlock gets smaller and smaller, until $\kappa$ reaches a minimum of $0.5$.
+
+\subsubsection{Staking rate \& real yield}
+
+The \textit{staking rate} is the percentage of tokens staked out of the total circulating supply in a given period. Other protocols refer to this variable as the `participation rate' or `total staked'. The \textit{real yield} is used to describe the true subdisy earnings of actively staking service-providers at a given timestamp, accounting for the staking rate and dilution. The real yield is a function of the nominal issuance (or `inflation rate' in other networks). The real yield is usually expressed as a percentage of stake size, and on an annual basis. Other protocols refer to this variable as the `adjusted reward' or simply `yield'. 
+\\\\
+The NuCypher protocol distributes subsidies such that the real yield $Y_r$ in a given period is inversely proportional to to the staking rate $\lambda$: 
+
+\begin{equation}
+    Y_r \:\:\alpha\:\: \frac{I_{max}}{\lambda}
+\end{equation}
+
+For example, the calculations for figures 4 and 5 assume a constant staking rate of 70\%.
+
+\section{Empirical analysis of existing networks}
+
+\textit{Full study to be published. This section summarizes analysis thus far and evaluates results with respect to the two-phase model.}
+
+\subsection{Outline}
+
+\subsubsection{Objectives \& Data Characteristics}
+
+The objective of this analysis is to better understand the relationship, if any, between (1) the variable real value of a subsidy received by service-providers/stakers (the \textit{real yield}), and (2) their subsequent, collective decision to increase, decrease or maintain the size of their stake, expressed as a percentage of the total circulating tokens (the \textit{staking rate}). This pertains directly to the primary objective of the NuCypher staking protocol mentioned in section 1 – to maximise the abundance and reliability of network supply – which is reflected in the health\footnote{`Health' in this context simply means the percentage of tokens staked – for example, a network with only 5\% of tokens staked might be described as unhealthy, because it may be highly centralized and/or not have the capacity to deal with demand} and stability of the staking rate. This study employs data sets from two independent sources, (A) Staking Rewards and (B) Yields:
```

Comment:
footnote numbers reset to 1 here in the pdf

Reply:
Eagle-eyed! I'll fix this